crates/bifrost/src/providers/replicated_loglet/loglet.rs

```diff
@@ -151,6 +152,19 @@ impl<T: TransportConnect> Loglet for ReplicatedLoglet<T> {
     }
 
     async fn enqueue_batch(&self, payloads: Arc<[Record]>) -> Result<LogletCommit, OperationError> {
+        let labels = [
+            ("loglet_id", self.my_params.loglet_id.to_string()),
+            ("node_id", self.networking.my_node_id().to_string()),
+        ];
```

Comment:
I'm a little worried about the cardinality of those metrics that use these labels. Maybe avoid them until we have a mechanism to switch on/off metrics by some sort of debugging level.

Reply:
Okay